Gemfile

```diff
@@ -55,6 +57,9 @@ end
 group :test do
   # Adds support for Capybara system testing and selenium driver
   gem 'capybara', '>= 2.15'
+  gem 'poltergeist'
+  gem 'launchy'
+  gem 'database_cleaner'
```

Comment:
There are probably some gems that can be removed that are included by default in Rails

Reply:
Commented out tzinfo to stop getting the stupid error.